aten/src/ATen/native/mkldnn/Utils.h

```diff
@@ -25,8 +25,18 @@ std::vector<int64_t> pool_output_sizes(
 };
 
 inline bool mkldnn_bf16_device_check() {
-  return cpuinfo_initialize() && cpuinfo_has_x86_avx512bw()
-      && cpuinfo_has_x86_avx512vl() && cpuinfo_has_x86_avx512dq();
+  return cpuinfo_initialize() && ((cpuinfo_has_x86_avx512bw()
+     && cpuinfo_has_x86_avx512vl() && cpuinfo_has_x86_avx512dq()) || (cpuinfo_has_arm_bf16()));
```

Comment:
this gets compiled in an environment where cpuinfo_has_arm_bf16 is not available. I need to revert this, let's sync up about what the proper fix is.